package.json

```diff
@@ -16,18 +16,24 @@
     "@restless/sanitizers": "^0.2.5"
   },
   "devDependencies": {
+    "@babel/core": "^7.0.0-0",
+    "@babel/plugin-syntax-flow": "^7.14.5",
+    "@babel/plugin-transform-react-jsx": "^7.14.9",
```

Comment:
why do we need babel?

Reply:
peer dependencies requirement

Reply:
From which package? :)

Reply:
`eslint-config-react-app`

Reply:
.. but now I realized we don't use it in this scope, so removed it! :) good catch!